Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM node:lts-alpine as builder
+
+# Install dependencies
+RUN apk add --no-cache git
+
+# Initialize the working directory
+WORKDIR /app
+
+# Copy package.json
+COPY package*.json ./
+
+# Install dependencies
+RUN npm install
+
+# Copy the source code
+COPY . .
+
+# Prune to production dependencies
+RUN npm prune --production
```

Comment:
You can just do `RUN npm install --production` above instead

Reply:
somehow it saved an extra megabyte.  Without prune the final image was 152MB. With prune it went to 151MB